shared/version/flex.go

```diff
@@ -17,6 +17,84 @@ var UserAgent = getUserAgent()
 // APIVersion contains the API base version. Only bumped for backward incompatible changes.
 var APIVersion = "1.0"
 
+// APIExtensions is the list of all API extensions in the order they were added.
```

Comment:
We should probably move things around a bit in this package, having three files at least:
 - flex.go for Version variable
 - useragent.go for the UserAgent variable and getUserAgent
 - api.go for APIVersion and APIExtensions

Reply:
Fixed in b9129e1876c22ecc4755d3c200d3a805653fbbbb.